Algorithm.CSharp/RawPricesCoarseUniverseAlgorithm.cs

```diff
@@ -0,0 +1,94 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using QuantConnect.Data.Market;
+using QuantConnect.Data.UniverseSelection;
+using QuantConnect.Orders;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    /// <summary>
+    /// In this algorithm we demonstrate how to use the coarse fundamental data to
+    /// define a universe as the top dollar volume and set the algorithm to use
+    /// raw prices
+    /// </summary>
+    /// <meta name="tag" content="using data" />
+    /// <meta name="tag" content="universes" />
+    /// <meta name="tag" content="coarse universes" />
+    /// <meta name="tag" content="regression test" />
+    public class RawPricesCoarseUniverseAlgorithm : QCAlgorithm
+    {
+        private const int NumberOfSymbols = 5;
+
+        public override void Initialize()
+        {
+            SetStartDate(2014, 01, 01);
+            SetEndDate(2015, 01, 01);
+            SetCash(50000);
+
+            // what resolution should the data *added* to the universe be?
+            UniverseSettings.Resolution = Resolution.Daily;
+
+            // Set the security initializer to set the data normalization mode to raw
+            SetSecurityInitializer(x => x.SetDataNormalizationMode(DataNormalizationMode.Raw));
```

Comment:
Can you please expand this a little with `{ }` to do a second operation - say set `ConstantFeeModel(0)` ? Lamba expressions aren't "beginner" level I'm hoping for with the example strategies.